src/Result.php

```diff
@@ -29,22 +29,25 @@ public function fetchAssociative()
 
     public function fetchOne()
     {
-        return $this->connection->fetchRow($this->result)[0];
+        $result = $this->connection->fetchRow($this->result);
+        return $result ? $result[0] : false;
```

Comment:
```suggestion
        return $result ? $result[0] : null;
```

Reply:
FetchOne is expected to send false when no value is found and not “null”

Reply:
![387211B4-D532-46F3-B191-1AA61A4DC296](https://user-images.githubusercontent.com/7591484/140746077-9636f298-9571-4fca-ab7e-76f42d4e9bff.jpeg)